src/Security/Voter/CitizenProject/CreateCitizenProjectVoter.php

```diff
@@ -27,10 +26,9 @@ protected function supports($attribute, $subject)
 
     protected function doVoteOnAttribute(string $attribute, Adherent $adherent, $subject): bool
     {
-        // Cannot create a project when already administrate one
+        // Cannot create a project when already has three citizen projects
         return $adherent->isAdherent()
-            && !$adherent->isCitizenProjectAdministrator()
```

Comment:
are you sure for this deletion?

Reply:
Yes, adherent can create/have max 3 CP with any status, and so he can be an administrator of several CP.